scripts/just.config.ts

```diff
@@ -116,7 +120,7 @@ export function preset() {
       'sass',
       'ts',
       // v9 needs to run api-extractor which generates rolluped .d.ts files that are shipped to npm
-      condition('api-extractor', () => isConvergedPackage() || !args.min),
```

Comment:
I'm guessing `args.min` was only used in one place ?

Reply:
yeah